impl/src/main/java/org/jboss/arquillian/graphene/GrapheneElementImpl.java

```diff
@@ -269,39 +269,20 @@ public Coordinates getCoordinates() {
         return element.getScreenshotAs(outputType);
     }
 
-    /*
-     * (non-Javadoc)
-     *
-     * @see java.lang.Object#hashCode()
-     */
     @Override
     public int hashCode() {
-        final int prime = 31;
-        int result = 1;
-        result = prime * result + ((element == null) ? 0 : element.hashCode());
-        return result;
+        // see #equals
+        return element.hashCode();
```

Comment:
What I'm missing here is the check if the element is null. I know that the probability is very low, but still...